examples/source-to-knative-service/README.md

```diff
@@ -98,8 +98,10 @@ kubectl create clusterrolebinding default-admin \
   --clusterrole=cluster-admin \
   --serviceaccount=default:default
 
+KAPP_CONTROLLER_VERSION=0.27.0
```

Comment:
imo the biggest benefit of this pattern is that it then becomes really easy to find everywhere where the version of a particular dependency exists:

```bash
ag KAPP_CONTROLLER_VERSION
```

and then boom, you found all the places:

```
site/content/docs/v0.0.6/install.md
222:KAPP_CONTROLLER_VERSION=0.27.0
225:  -f https://github.com/vmware-tanzu/carvel-kapp-controller/releases/download/v$KAPP_CONTROLLER_VERSION/release.yml

site/content/docs/install.md
222:KAPP_CONTROLLER_VERSION=0.27.0
225:  -f https://github.com/vmware-tanzu/carvel-kapp-controller/releases/download/v$KAPP_CONTROLLER_VERSION/release.yml

hack/setup.sh
34:readonly KAPP_CONTROLLER_VERSION=0.27.0
224:                -f https://github.com/vmware-tanzu/carvel-kapp-controller/releases/download/v$KAPP_CONTROLLER_VERSION/release.yml |

examples/source-to-knative-service/README.md
101:KAPP_CONTROLLER_VERSION=0.27.0
104:    -f https://github.com/vmware-tanzu/carvel-kapp-controller/releases/download/v$KAPP_CONTROLLER_VERSION/release.yml
```